content/html/concepts/elements/terms/input/input.md

```diff
@@ -44,23 +44,23 @@ Some common attributes used to set properties for the `<input>` element are show
 
 ## Example
 
-The following examples uses a `<form>` element that features `<input>` elements:
+The following examples uses a `<form>` element that features `<input>` elements (Also, the `for` attributes are linking the [<label> elements](https://www.codecademy.com/resources/docs/html/elements/label) to the `id` of the <input> elements.):
```

Comment:
I think this additional sentence can be taken out of the parentheses. Also, do we want to mention explicit vs implicit association of the label and input tags?

In the example below, the inputs are following the label tag. If we nested the inputs in the label, then they would be associated even without the for/id attributes.

Reply:
Interesting question @yangc95. I think mention the explicit vs implicit association would encourage students to consider accessibility when making forms, a topic that I've noticed is not prioritized enough in conversations about learning how to code.